### ISSUE TITLE
URL's metatags are relative, s/b absolute — in production

### ISSUE THREAD

**Issue by h-enk** [h-enk/doks#213]
## Description

URL's metatags are relative, s/b absolute — in non development environments.

### Steps to reproduce

1. Run `npm run build`
2. View page source — e.g.:

```html
<meta property="og:url" content="/"><meta property="og:image" content="/doks.png">
```

See also e.g. [doks.netlify.app](https://doks.netlify.app/)

### Expected result

```html
<meta property="og:url" content="https://doks.netlify.app/"><meta property="og:image" content="https://doks.netlify.app/doks.png">
```

### Actual result

```html
<meta property="og:url" content="/"><meta property="og:image" content="/doks.png">
```

### Environment

```bash
> @hyas/doks@0.2.3 precheck
> npm version

{
  '@hyas/doks': '0.2.3',
  npm: '7.0.5',
  node: '15.9.0',
  v8: '8.6.395.17-node.25',
  uv: '1.41.0',
  zlib: '1.2.11',
  brotli: '1.0.9',
  ares: '1.17.1',
  modules: '88',
  nghttp2: '1.42.0',
  napi: '7',
  llhttp: '2.1.3',
  openssl: '1.1.1i',
  cldr: '38.1',
  icu: '68.2',
  tz: '2020d',
  unicode: '13.0'
}

> @hyas/doks@0.2.3 check
> hugo version

hugo v0.82.0-9D960784+extended windows/amd64 BuildDate=2021-03-21T17:28:04Z VendorInfo=gohugoio
```

**Comment by thrau** [comment]
The same issue applies to twitter cards, which causes the social preview cards to not render the image, they need an absolute URl. We solved it by using `absURL` everywhere, is this an option? For example here:
https://github.com/h-enk/doks/blob/cd652fc7face596f1486ee3a87d517fc3ef2979f/layouts/partials/head/opengraph.html#L11-L28

We would just use `<meta property="og:image" content="{{ $.RelPermalink | absURL }}{{ . }}"> `, etc.

I'm new to doks and hugo and don't know whether this is the correct way of doing it.

**Comment by h-enk** [comment]
Fixed w/ 819cbc3ae8408cd1bf195149f9290542d90cb5e4
